### ISSUE TITLE
Hard Error on npm >5

### ISSUE THREAD

**Issue by gsklee** [react-community/create-react-native-app#275]
I think this is related to https://github.com/expo/xde/issues/121, however the error message is slightly different so I am not sure (it didn't ask me to run `npm i -g npm@4.6.1`):

```
$ yarn start                    
yarn start v0.24.6
$ react-native-scripts start 
09:39:51: Starting packager...
***ERROR STARTING PACKAGER***

Error: You are using npm version 5.0.3. Please use an npm version that is >= 3.0.0 and < 5.0.0.
09:39:52: Error starting packager: Error: Couldn't start project. Please fix the errors and restart the project.
    at /xdl/src/Project.js:1038:11
    at Generator.next (<anonymous>)
    at step (/Users/gsklee/Projects/xxx/xxx-app/node_modules/xdl/build/Project.js:1468:191)
    at /Users/gsklee/Projects/xxx/xxx-app/node_modules/xdl/build/Project.js:1468:361
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
error Command failed with exit code 1.
```

**Comment by gsklee** [comment]
Okay, they are separated issues; I rolled back to my previous `yarn.lock`, and now I can start up my project using `react-native-scripts start` while the XDE still gives the error reported inside https://github.com/expo/xde/issues/121

**Comment by HermantoTang** [comment]
@gsklee have you fixed this issue? I got the same error after running
```
create-react-native-app myApp
cd myApp
yarn start
```

**Comment by gsklee** [comment]
@HermantoTang Nope, I rolled back to an earlier `yarn.lock` file I had; if you're starting a new project I guess you'll have to wait. Apparently some dependency further up is causing the problem but no idea which one right now.

**Comment by vikas199** [comment]
i am enable to create react native app using npm 5.0.3. Any suggestions

**Comment by brentvatne** [comment]
this is resolved for npm > 5.0.3. create-react-native-app does not work at all for npm 5.0.0 - 5.0.3 because it is too buggy, please update if you are using it.
